docs/source/example_notebooks/gcm_linear_scm_bootstap_pvalues/false_discovery_rate_control.py

```diff
@@ -0,0 +1,152 @@
+     however, this does not allow for missing values which could occure is using semopy to estimate an SEM with a latent node
+
+    :param unajusted_p_values: An array or a list of p-values. [could include NANs]
+    
+    :return: Adjusted FDR p-Values in same order as the input pvaules. 
+
+    """
+
+    mp=len(unajusted_p_values)
+    p_valsArr = np.array(unajusted_p_values)
+
+    #before starting ensure numeric values and just count them - i.e. exlude null values
+    m=0
+    p_vals = np.empty(mp)
+    for i in range(mp):
+        p_vals[i]=numberfy(p_valsArr[i])
+        if p_vals[i] >= 0:
+            m=m+1
+        
+    #calcualte FDR adjusted pvalues
+    ranked_p_values = rankdata(p_vals)
+    adjfdr = p_vals * m / ranked_p_values
+    adjfdr[adjfdr > 1] = 1
+    
+    #rememebr if (k) is rejected (1)..(k) are rejected - the FDR adjustemnt shoud reflect that
+    IndexRanks = p_vals.argsort()
+    minAP=1
+    for i in range(mp-1,-1,-1):
+        pointer = int(IndexRanks[i])
+        if adjfdr[pointer] > minAP:
+            adjfdr [pointer] = minAP
+        else:
+            minAP = adjfdr[pointer]
+    return adjfdr
+
+
+def adjusted_FDCR_pvalues(unajusted_p_values,
+                          belife_scores,
+                          intersection_belife_score=10000.0
+                          )-> np.ndarray:
+    """ Adjusts p-values for FDR using the Benjamini-Kling Weighted FDCR control adding the Simes family statistic
+    
+    Benjamini Y., Hochberg Y. (1995). Controlling the false discovery rate: a practical and powerful approach to multiple testing, Journal of the Royal Statistical Society B, 289-300.
+    Benjamini, Y., Hochberg, Y. (1997). "Multiple hypotheses testing with weights", Scandinavian Journal of Statistics, vol 24, 3, pp 407-418.
+    Benjamini Y, Kling Y. E. (2005) A Cost-Base approach to Multiplicity Control, technical paper 
+        [can be downloaded at https://docs.google.com/a/businessken.com/viewer?a=v&pid=sites&srcid=YnVzaW5lc3NrZW4uY29tfGhvbWV8Z3g6MTMwODI4YmVmNjI5ZTc4Yg ]
+
+    the convertion of the belife scores to costs as per the original paper above is chosed to be the reciprocal.
+
+    :param unajusted_p_values: An array or a list of p-values. [could include NANs]
+    :param belife_scores: - vector of, non negaive, Belife Scores coresponding to the p-values
+                             Inf=the SME thinks the null hypotheiss is absolutly wrong = there is definatly somthing in it,
+                             0 = the SME belives the null hypotheis is an absolute truth - there is nothing to this claim
+                             thus the costs are the inverse of the belife
+                   these are relative belifes. if all are equal then we are back to the regular BH procedure [with the adition of the Simes statistic]
+                   assume len(unajusted_p_values) = len(belife_scores) and the elemets are coresponding
```

Comment:
There are some typos in the docstring and variable names, can you double check that?